packages/editor-ui/src/Interface.ts

```diff
@@ -842,3 +842,8 @@ export interface IBounds {
 	maxX: number;
 	maxY: number;
 }
+
+export interface IDebounce {
```

Comment:
is this type really needed? seems like setting it at function level is enough. I mean we are only using it in one place

Reply:
Removed.